### PR TITLE
Check for buffer

### PR DESCRIPTION
**Pull request by onmyway133**
Or this error may show up `+[AVCaptureStillImageOutput jpegStillImageNSDataRepresentation:] - NULL sample buffer.`

### CODE REVIEW

File: Source/CameraView/CameraView.swift [hunk 1]
```diff
@@ -264,9 +264,11 @@ class CameraView: UIViewController, CLLocationManagerDelegate {
     dispatch_async(queue, { [unowned self] in
       stillImageOutput.captureStillImageAsynchronouslyFromConnection(stillImageOutput.connectionWithMediaType(AVMediaTypeVideo),
         completionHandler: { (buffer, error) -> Void in
-          let imageData = AVCaptureStillImageOutput.jpegStillImageNSDataRepresentation(buffer)
 
-          guard let imageFromData = UIImage(data: imageData) else { return }
+          guard error == nil && buffer != nil && CMSampleBufferIsValid(buffer),
```
Comment (zenangst) [inline Source/CameraView/CameraView.swift]: giphy-60

Comment (zenangst) [inline Source/CameraView/CameraView.swift]: Do you need to check for not `nil` when you rely on `CMSampleBufferIsValid`?

Comment (onmyway133) [inline Source/CameraView/CameraView.swift]: @zenangst just to be sure, because it is implicitly unwrapped optional 😬

Comment (zenangst) [inline Source/CameraView/CameraView.swift]: Oh I see;

```swift

func CMSampleBufferIsValid(_ sbuf: CMSampleBuffer!) -> Boolean
```

**Pull request merged**